src/sass/components/_button.scss

```diff
@@ -34,3 +34,12 @@
     color: #ffffff;
   }
 }
+.status-buttons-active {
```

Comment:
Just suggestion: instead of creating the new custom class name, you could use a combination of a parent(s) and common `active` name, ie. `.actions .btn-outline.active` (please notice, there is no space between `btn-outline` and `active` classes).